bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/MagnitudeSystem.scala

```diff
@@ -20,8 +20,8 @@ sealed abstract class SurfaceBrightness(val name: String) extends BrightnessUnit
   */
 object MagnitudeSystem {
 
-  case object Vega                extends MagnitudeSystem("Vega")
-  case object AB                  extends MagnitudeSystem("AB")
+  case object Vega                extends MagnitudeSystem("Vega mag")
```

Comment:
I thin this used when displaying the Mag systems in the OT. is this change desired?